options.go

```diff
@@ -106,6 +106,25 @@ func OptHTTPLogger(l *log.Logger) Option {
 	}
 }
 
+// OptEnableCustomRoutePathPrefix enables custom routes in the server that
+// start with the given prefix. A user must also provide an API
+// prefix in this case and the two must not overlap. Otherwise,
+// the configuration will be rejected.
+func OptEnableCustomRoutePathPrefix(prefix string) Option {
```

Comment:
Validation. Prefix must start with a slash, and not be terminated by a /

Reply:
yes. done